### ISSUE TITLE
Make suspend inhibition optional

### ISSUE THREAD

**Issue by kristozondagh** [rafaelmardojai/blanket#341]
Since the last update my PC not longer suspends unless I close Blanket. 

I'm running the latest Flatpak on Fedora 39.

**Comment by rafaelmardojai** [comment]
Yeah, It now inhibits suspend on playback, but if is doing the same when paused then there's a bug.

**Comment by mabequinho** [comment]
I'm having the same issue while blanket is paused, and when combined with Gnome Break Timer.
Break timer forces a screen lock -> Blanket is paused immediately -> Computer never suspends.

**Comment by lstrtstr** [comment]
> It now inhibits suspend on playback

Could this please be optional? Some setting to disable inhibition?

**Comment by mabequinho** [comment]
> > It now inhibits suspend on playback
> 
> Could this please be optional? Some setting to disable inhibition?

![image](https://github.com/rafaelmardojai/blanket/assets/46428150/21911ff4-a7d4-4e9c-a147-1db85866df07)

**Comment by lstrtstr** [comment]
> For the flatpak version you can remove mpris support.

Which is not a good workaround at all, as it also removes Blanket from playerctl.
I want to control Blanket via playerctl, but I don't want Blanket to inhibit suspend.

**Comment by develop7** [comment]
Not only that, having Blanket running hides suspend-related settings in GNOME's Power settings (I admit it could be GNOME's bug, but still). Looks tad bit too intrusive to me, would appreciate cranking it down a little.

**Comment by rafaelmardojai** [comment]
> Not only that, having Blanket running hides suspend-related settings in GNOME's Power settings (I admit it could be GNOME's bug, but still). Looks tad bit too intrusive to me, would appreciate cranking it down a little.

That indeed seems like a weird behavior, but is how GNOME behaves when there's an app inhibiting suspension.
